nservicebus/security/encryption.md

```diff
@@ -47,8 +52,15 @@ Property encryption is enabled via the configuration API.
 
 <!-- import EncryptionServiceSimple -->
 
+### Generating encryption keys
+
+In order to use the RijndaelEncryptionService a primary and secondary keys need to be generated. An option is to use `RijndaelManaged.GenerateKey`, please refer to the following article to generate new keys:
+
+- https://msdn.microsoft.com/en-us/library/system.security.cryptography.rijndaelmanaged.generatekey(v=vs.110).aspx
```

Comment:
There are many snippets that have no dependency on NSB. The value is having them compilable